src/api/app/models/comment.rb

```diff
@@ -9,6 +9,10 @@ class Comment < ApplicationRecord
 
```

Comment:
regarding the commit message... There is a typo in the first sentence "the the class". And a comma in the second sentence would be nice:-)